disasm/disasm.cc

```diff
@@ -1822,6 +1822,15 @@ disassembler_t::disassembler_t(int xlen)
     DEFINE_RTYPE(pktt32);
   }
 
+    // ext-cmo
+  DISASM_INSN("cbo.clean", cbo_clean, 0, {&xrs1});
+  DISASM_INSN("cbo.flush", cbo_flush, 0, {&xrs1});
+  DISASM_INSN("cbo.inval", cbo_inval, 0, {&xrs1});
+  DISASM_INSN("cbo.zero", cbo_zero, 0, {&xrs1});
+  DISASM_INSN("prefetch.i", prefetch_i, 0, {&store_address});
+  DISASM_INSN("prefetch.r", prefetch_i, 0, {&store_address});
```

Comment:
`prefetch_i` → `prefetch_r`

Reply:
Sorry. I forgot to update the second argument after copy.